### PR TITLE
Update description for odo app list

### PR DESCRIPTION
**Pull request by cdrage**
Modifies the help description to keep in-line with `odo list`

Closes: https://github.com/redhat-developer/odo/issues/644

### CODE REVIEW

File: cmd/application.go [hunk 1]
```diff
@@ -130,9 +130,9 @@ var applicationDeleteCmd = &cobra.Command{
 
 var applicationListCmd = &cobra.Command{
 	Use:   "list",
-	Short: "Lists all the applications",
-	Long:  "Lists all the applications",
-	Example: `  # List all applications
+	Short: "List all applications in the current application",
```
Comment (piyush1594) [inline cmd/application.go]: It should be in `current project`. BTW, my PR #684 will also fix this

**Comment by surajnarwade** [comment]
@cdrage, Since we are planning to provide global flags for `project`, `application` and `component`, it will be possible to list app, component of any project or application

Reference: https://github.com/redhat-developer/odo/pull/683

**Comment by cdrage** [comment]
Updated the PR. If we can merge this it'll fix: https://github.com/redhat-developer/odo/issues/644 although I do know that #683 will be changing this in the future!

**Comment by cdrage** [comment]
This PR keeps failing tests for some reason. Super flakey. Do you know what's up @ashetty1 ?

**Comment by piyush1594** [comment]
@cdrage You can merge this PR

**Pull request merged**